### ISSUE TITLE
Ctrl + Shift + I doesn't work anymore

### ISSUE THREAD

**Issue by Georgewxm** [microsoft/vscode#232188]
Type: <b>Bug</b>

When I press "Ctrl + Shift + I" to bring out the chat window, its responses no longer looks at the context of my code. Selecting also does not get picked up by copilot either

Extension version: 0.21.2
VS Code version: Code - Insiders 1.95.0-insider (804f450ca900d24db25e7174e8b6dfb3fb2a318c, 2024-10-22T13:30:10.100Z)
OS version: Windows_NT x64 10.0.22631
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 9 7945HX with Radeon Graphics         (32 x 2495)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|31.69GB (16.15GB free)|
|Process Argv|--crash-reporter-id bc8ceae5-6b7c-4e0a-9d4e-8cc10d13fa7f|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805:30301674
vsaa593cf:30376535
py29gd2263:31024238
c4g48928:30535728
962ge761:30841072
pythongtdpath:30726887
pythonnoceb:30776497
asynctok:30898717
dsvsc014:30777825
dsvsc015:30821418
pythonmypyd1:30859725
h48ei257:31000450
pythontbext0:30879054
cppperfnew:30980852
pythonait:30973460
01bff139:31013167
dvdeprecation:31040973
dwnewjupytercf:31046870
nb_pri_only:31057983
nativerepl1:31134653
refactort:31084545
pythonrstrctxt:31093868
wkspc-onlycs-t:31132770
nativeloc1:31118317
wkspc-ranged-t:31151552
cf971741:31144450
e80f6927:31120813
defaultse:31146405
12bdf347:31141542
iacca1:31150324
notype1:31143044
dwcopilot:31158714
g7688163:31155431

```

</details>

**Comment by laszukdawid** [comment]
@TylerLeonhardt when can we expect that to be shipped?

**Comment by TylerLeonhardt** [comment]
Should be shipped already in 1.95 of VS Code. Note, the Quick Chat keybinding has changed... it's Ctrl+Alt+Shift+L.
